cmake/FindCaDiCaL.cmake

```diff
@@ -1,27 +1,99 @@
 #####################
-## FindCaDiCaL.cmake
-## Top contributors (to current version):
-##   Mathias Preiner
-## This file is part of the CVC4 project.
-## Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
-## in the top-level source directory and their institutional affiliations.
-## All rights reserved.  See the file COPYING in the top-level source
-## directory for licensing information.
-##
+# FindCaDiCaL.cmake
+# Top contributors (to current version):
+#   Mathias Preiner
+# This file is part of the CVC4 project.
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
```

Comment:
Oh, this change will break the update copyright script as it expects `##`. Can you revert the change?

Reply:
done,